### ISSUE TITLE
[Question] What will be updated in prisma-nexus when prisma2 is released?

### ISSUE THREAD

**Issue by hyochan** [graphql-nexus/nexus-plugin-prisma#296]
Hi! Today, I've been working hard in making production server with `prisma-nexus` and I really love it. My question is what's the effect with `prisma-nexus` when `prisma2` is released?

I am concerned with the migration.

Thanks.

**Comment by darrylyoung** [comment]
I’d also be interested to know. I’m aware that the Prisma 2 preview was only released today but I’ve not seen any mention of Nexus. I’m looking forward to seeing where this all goes but, like you, I just finished migrating my app to Nexus so I’d like to know what’s coming.

**Comment by hyochan** [comment]
Prisma2 is having `nexus-prisma` internally in boilerplate.
![image](https://user-images.githubusercontent.com/27461460/59753980-33af1880-92c0-11e9-961e-581380d44476.png)


Love to see how we can migrate our `nexus-prisma` project.

**Comment by divyenduz** [comment]
Thanks everyone for participating in this discussion! There would be some breaking API changes in nexus with Prisma 2. @hyochan's comment points to how that would look like, we also rewrote our [typescript](https://github.com/prisma/photonjs/tree/master/examples/typescript) and [javascript](https://github.com/prisma/photonjs/tree/master/examples/javascript) examples with the upcoming API. Please take a look and share your feedback. 

These changes are documented here as well: https://nexus.js.org/docs/database-access-with-prisma-v2

While there are some changes in the API, the new API is much shorter and migration should be straight forward.

**Comment by pierreyves-lebrun** [comment]
> I’d also be interested to know. I’m aware that the Prisma 2 preview was only released today but I’ve not seen any mention of Nexus. I’m looking forward to seeing where this all goes but, like you, I just finished migrating my app to Nexus so I’d like to know what’s coming.

I really love Prisma but I have to say that working on a project with the threat that it might be using obsolete/deprecated tools in less than 6 month time does concern me.

Anyway, I guess it's more fun that way! :-)

**Comment by darrylyoung** [comment]
> I really love Prisma but I have to say that working on a project with the threat that it might be using obsolete/deprecated tools in less than 6 month time does concern me.

@pierreyves-lebrun Yeah, I know what you mean. I know there's no _need_ to always upgrade to the latest and greatest but I've personally found Prisma to be moving so fast that before I've finished building something, there's often a much _better_ way to do it, e.g. with the recent introduction of nexus, which I just upgraded my app to.

Anyway, progress is good, I guess. I'm looking forward to seeing where they go with Prisma 2!

**Comment by jasonkuhrt** [comment]
Related to #421
